### PR TITLE
feat: [HL-4] Update label styles 🎯

### PR DESCRIPTION
**Pull request by abdallah-7698**
Change the labels for new Design (Food Recipe App)

### CODE REVIEW

**Review by Ahattalla — APPROVED** [review]

File: Healthy/Classes/Extensions/UILabel+Style.swift [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
+    
     func applyStyle(_ style: Style) {
         switch style {
-        case .onboardingTitle:
-            applyOnboardingTitleLabelStyle()
-        case .onboardingSubtitle:
-            applyOnboardingSubtitleLabelStyle()
-        case .descriptionLabel:
-            applyDescriptionLabelStyle()
+        case .splashHeaderLabel:
```
Comment (abdelhamid-f-nasser) [inline Healthy/Classes/Extensions/UILabel+Style.swift]: I think we can move the `applyStyle` method inside the enum so we can just call it statically with the type.
example: `Style.onboardingTitle.applyStyle`

File: Healthy/Classes/Extensions/UILabel+Style.swift [hunk 2]
```diff
@@ -1,26 +1,38 @@
 import UIKit
 
 extension UILabel {
-
+    
     // MARK: Style
-
+    
     enum Style {
```
Comment (ahmdmhasn) [inline Healthy/Classes/Extensions/UILabel+Style.swift:7]: Style should not be aware of the context it applies to.

Comment (abdallah-7698) [reply Healthy/Classes/Extensions/UILabel+Style.swift:7]: 👍

File: Healthy/Classes/Extensions/UILabel+Style.swift (hunk 1 above)
Comment (abdallah-7698) [reply Healthy/Classes/Extensions/UILabel+Style.swift]: but if move `applyStyle` to the enum i will not be able to access the label color or label font and I got the `error : Instance member 'applySplashHeaderLabelStyle' of type 'UILabel' cannot be used on instance of nested type 'UILabel.Style'`

**Review by ahmdmhasn — APPROVED** [review]

**Pull request merged**